bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -228,6 +228,14 @@ This certificate is used for pairing between the Bridge and the Bosch Smart Home
 
 _Press and hold the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing_.
 
+### Supported Channels
```

Comment:
I'm wondering whether we should add _Smart Home Controller_  to the list of  _Supported Things_ and add the channels there instead of the _Bridge Configuration_ section.

Reply:
Initially I added the channel description to the Bridge because thats where I implemented them. But I am also fine putting it to the _Supported Things_ section.
Is there any specific order where I should put the new _Smart Home Controller_ section?

Reply:
I would propose to add it as the first thing in the `Supported Things` section.